QMLComponents/controls/jaspdoublevalidator.cpp

```diff
@@ -31,28 +32,34 @@ QValidator::State JASPDoubleValidator::validate(QString& s, int& pos) const
 	if (s.startsWith("-") && bottom() >= 0)
 		return QValidator::Invalid; 
 	
+	// check range of value
+	double value;
+	bool isNumber	= QColumnUtils::getDoubleValue(	s, value);
+	//int intVal;
+	//bool isInt		= QColumnUtils::getIntValue(	s, intVal);
+	
+	//Maybe the number is formatted in some crazy way, because of locales
+	QString toEnglish = isNumber ? QString::number(value) : s;
+	
 	// check length of decimal places
-    QString point = locale().decimalPoint();
-	int indexPoint = s.indexOf(point);
+    QString point = ".";
+	int indexPoint = toEnglish.indexOf(point);
```

Comment:
Apparently this does not work: for an IntegerField you can now only have a number of 6 digits. And for a DoubleField you can have only 6 digits before the decimal sign.